lib/utils/notifications.dart

```diff
@@ -120,3 +125,48 @@ Future<void> pollRepliesAndShowNotifications() async {
   // Save our poll time
   prefs.setString(_lastPollTimeId, DateTime.now().toString());
 }
+
+void navigateToNotificationReplyPage(BuildContext context, {required int? replyId}) async {

```

Comment:
Would it be good to place this in a separate file? Similar to `navigate_comment.dart` or `navigate_instance.dart`?

Reply:
Sure! Done!